modules/standard/Subprocess.chpl

```diff
@@ -584,7 +584,7 @@ module Subprocess {
       // goes out of scope, but the channel will still keep
       // the file alive by referring to it.
       try {
-        var stdin_file = openfd(stdin_fd, hints=QIO_HINT_OWNED);
+        var stdin_file = openfd(stdin_fd, hints=ioHintSet.direct(QIO_HINT_OWNED));
```

Comment:
I think this is okay for now, but it might suggest adding an explicit type method to `ioHintSet` to support it more naturally - I'd open an issue for each one you're calling `.direct` with, to discuss having a user-facing version